tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationIgnore/common-Translations.ignore

```diff
@@ -10,12 +8,11 @@ E0181
 E0182
 E0183
 E0184
-E0185
 E0186
 E0187
 E7069
 E7070
 E7071
 InvalidFramework
 InvalidPlatform
-E7072
+E7072
```

Comment:
Missing end-of-line

Reply:
Good catch! I think this file and the reliance on it will become obsolete after we get OneLocBuild going?